automation_tools/__init__.py

```diff
@@ -237,10 +237,13 @@ def setup_avahi_discovery():
        by 'ping vm.local' run at Satellite
     """
     os_version = distro_info()[1]
-    run('yum -y install https://dl.fedoraproject.org/pub/epel/'
-        'epel-release-latest-{0}.noarch.rpm'.format(os_version))
+    epel_was_missing = not run('rpm -qa epel-release')
```

Comment:
Note: rpm -qa epel-release
$?=0 always

Reply:
so no need for warn_only=True

Reply:
How about we do it this way?
```
epel_missing = run('rpm -qa epel-release')
if "epel-release" not in epel_missing:
     run('yum -y install https://dl.fedoraproject.org/pub/epel/'
           'epel-release-latest-{0}.noarch.rpm'.format(os_version))
```

Reply:
I've just realized that fabric supports ```return_code```
Using ```return_code``` it would be most precise.